wallet/subnet/primary/examples/register-l1-validator/main.go

```diff
@@ -137,5 +137,5 @@ func main() {
 	}
 
 	validationID := addressedCallPayload.ValidationID()
-	log.Printf("registered new subnet validator %s to subnet %s with txID %s as validationID %s in %s\n", nodeID, subnetID, registerSubnetValidatorTx.ID(), validationID, time.Since(registerSubnetValidatorStartTime))
+	log.Printf("registered new subnet validator %s to subnet %s with txID %s as validationID %s in %s\n", nodeID, subnetID, registerL1ValidatorTx.ID(), validationID, time.Since(registerL1ValidatorStartTime))
```

Comment:
```suggestion
	log.Printf("registered new L1 validator %s to subnetID %s with txID %s as validationID %s in %s\n", nodeID, subnetID, registerL1ValidatorTx.ID(), validationID, time.Since(registerL1ValidatorStartTime))
```

Reply:
Unfortunately not globally, since we still have the notion of "subnet validators" from the pre-existing (prior to Etna) `AddSubnetValidatorTx` and `RemoveSubnetValidatorTx`. 

There were a very additional instance of "subnet validator" that I found referring to L1 validators though, which I have corrected now.